index.js

```diff
@@ -26,10 +26,16 @@ module.exports = (function () {
    * @return {Promise} returns String passage
    */
   bible.get = function (psg, ver) {
-    if (typeof psg !== 'string') { throw new TypeError('Expected a string'); }
+    if (typeof psg !== 'string') {
+      throw new TypeError('Passage should be a string');
+    }
+    if (typeof ver !== 'undefined' && typeof ver !== 'string') {
+      throw new TypeError('Version should be a string');
+    }
 
     var self = this;
     var v = ver || 'asv';
```

Comment:
That was actually my initial solution, but I quickly realized that would not work since the purpose of the double pipes is to provide a default if `ver` is undefined. 

So if no value is passed for `ver` then we try to call a function on it we start throwing errors.

Another solution would be using a `if` `else` block instead of the pipes.